tools/SetupFlow/DevHome.SetupFlow/Views/QuickstartPlaygroundView.xaml

```diff
@@ -220,6 +220,16 @@
             </Grid>
 
             <!--Bottom half of UI-->
+            <Grid Grid.Row="2" Visibility="{x:Bind ViewModel.IsErrorViewVisible, Mode=OneWay}">
+                <StackPanel

```

Comment:
The icon above the text looks a little weird to me, should it move to be next to the TextBlocks? Also, suggest using TextWrapping="WrapWholeWords"

Reply:
That is an interesting thought about the icon placement.  Right now, it is place where design has suggested, but I can bring this up later with them and address it in a future PR if they don't disagree with it.  Will address WrapWholeWords.

Reply:
Does the guidance on wrapping whole words apply generally? (We have a few other uses in TextBoxes across this XAML; can / should those be corrected as well?)

Reply:
It is not supported with TextBox, but TextBlock does support it.  Addressed throughout: https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.textbox.textwrapping?view=windows-app-sdk-1.5#remarks